### ISSUE TITLE
Importing text fails sometimes

### ISSUE THREAD

**Issue by ozsay** [systemjs/systemjs#870]
http://plnkr.co/edit/Fpb9IRT6AcwPzWQRVDZn?p=preview

index.html
```
<!DOCTYPE html>
<html>
  <head>
    <script src="systemjs.js"></script>
    <script>
    System.config({
      map: {
        text: 'systemjs-text.js'
      }
    });
      var a = System.import('text1.txt!text');
      a.then(function(val) {
        console.log(val);
      });
      
      var b = System.import('text2.txt!text');
      b.then(function(val) {
        console.log(val);
      });
    </script>
    
  </head>

  <body>
    <h1>Hello Plunker!</h1>
  </body>

</html>
```

text1.txt
```
text1
```

text2.txt
```
var fakeImport = require('Not_really_a_require');
```

In the example I have 2 text imports. importing `text1.txt` is working fine but `text2.txt` fails.
It appears that somehow loader marks `Not_really_a_require` for loading (console shows a 404 error on `Not_really_a_require`). It happens even if the line is commented.

edit:
It looks like the loader (`cjs`) treats the text files as CommonJS files.
line for reference: https://github.com/systemjs/systemjs/blob/master/dist/system.src.js#L3475

(BTW I'm trying to import a script as a text).

Am I doing something wrong or is it a bug?

Thanks in advance,
Oz

**Comment by ozsay** [comment]
I solved it by changing the `plugin-text` code to this:
```
exports.instantiate = function(load) {
  return load.source;
};
```

plunker: http://plnkr.co/edit/TCcnwmSsCwLAGVjHRO7e?p=preview

**Comment by guybedford** [comment]
Thanks for posting! I believe I fixed this in https://github.com/systemjs/plugin-text/commit/7ef92bc6ac3b9714ef5d066e065034206f133007 in a way that will still support builds.

Let me know if that works out for you and I'll post a new release.

**Comment by ozsay** [comment]
Yes, It works. Both in the plunker and in my app.

Thanks.

**Comment by guybedford** [comment]
Thanks for the feedback - I've released a 0.0.3.
